uber/site_sections/preregistration.py

```diff
@@ -257,6 +257,7 @@ def delete(self, id):
     def dealer_confirmation(self, session, id):
         return {'group': session.group(id)}
 
+    @log_pageview
```

Comment:
I decided it'd be nice to see if an admin viewed ANY page where they could see or change attendee/group data. I think I got them all.